### PR TITLE
python: Revert #3153

### PR DESCRIPTION
**Pull request by alltilla**
We decided to keep the `bytes` type data.

Signed-off-by: Attila Szakacs <attila.szakacs@oneidentity.com>

### CODE REVIEW

**Comment by kira-syslogng** [comment]
Build SUCCESS

File: tests/functional/sngtestmod.py [hunk 1]
```diff
@@ -39,6 +39,8 @@ def is_open(self):
 
     def send(self, msg):
```
Comment (furiel) [inline tests/functional/sngtestmod.py]: Please remove the changenote as well.

File: tests/functional/sngtestmod.py [hunk 2]
```diff
@@ -39,6 +39,8 @@ def is_open(self):
...
         with open('test-python.log', 'a') as f:
+            if (isinstance(msg['MSG'], bytes)):
```
Comment (furiel) [inline tests/functional/sngtestmod.py]: Why do we need this check?

File: tests/functional/sngtestmod.py (hunk 1 above)
Comment (alltilla) [reply tests/functional/sngtestmod.py]: The NEWS file is already generated and formatted, and I removed this change from it.

File: tests/functional/sngtestmod.py (hunk 2 above)
Comment (alltilla) [reply tests/functional/sngtestmod.py]: I wanted to branch the `python2` from the `python3`, but yes, indeed it is not needed, as both of them use bytes... My bad.

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Pull request merged**